### PR TITLE
added code to show the path

### PR DESCRIPTION
**Pull request by srinivasmachiraju**
added this code to find the path for package_url
err

### CODE REVIEW

**Comment by srinivasmachiraju** [comment]
showed the path of the packageurl .py file in src folder

**Comment by aviral1701** [comment]
You should title it as : Add code to show the path instead of the above.Should be like you are giving order to the codebase.

**Comment by aviral1701** [comment]
Also which issue does it resolve?

**Comment by pombredanne** [comment]
Thank you! let me review

**Comment by pombredanne** [comment]
I need a signoff on all your commits

File: test_purl.py [hunk 1]
```diff
@@ -28,6 +28,10 @@
 import json
 import re
 import unittest
+import os
+src= os.path.abspath("packageurl.py")
+import sys
+sys.path.append("src")
```
Comment (pombredanne) [inline test_purl.py]: You never want to modify the sys.path in a library
Do you know about virtualenv?

**Comment by pombredanne** [comment]
We use a virtualenv and install packages as needed but do not mutate the sys.path... so I am closing this.

**Pull request closed without merging**